crates/madara/src/constants.rs

```diff
@@ -1,6 +1,7 @@
 // Madara constants
 pub const MADARA_DOCKER_IMAGE: &str = "madara";
 pub const MADARA_COMPOSE_FILE: &str = "compose.yaml";
+pub const MADARA_COMPOSE_FILE_CI: &str = "compose_ci.yaml";
```

Comment:
We should avoid having several compose files when possible to avoid synchronization issues between them.
Hopefully all custom configs can be managed at the .env level and then let the compose read those variables accordingly.